dotnet-download.ps1

```diff
@@ -0,0 +1,40 @@
+param(
+[parameter(Mandatory = $true)]
+$DOTNET_REMOTE_PATH,
+[parameter(Mandatory = $true)]
+$DOTNET_LOCAL_PATH,
+[parameter(Mandatory = $true)]
+$DOTNET_PATH)
```

Comment:
Looking around, normally they are actually PascalCase, I got that wrong. (Edited example.) Switching to PascalCase sounds great.